### PR TITLE
Update snyk file to reflect current vulnerabilities.

### PR DESCRIPTION
**Pull request by woodt**

### CODE REVIEW

File: .snyk [hunk 1]
```diff
@@ -1,10 +1,54 @@
...
+          as text fixtures (trusted source and execution context.)
+          Risk is acceptable.
+        expires: 2020-11-19T06:00:00.000Z
+
+  SNYK-PYTHON-SQLALCHEMY-173678:
+    - '*':
+        reason: >-
+          CKAN 2.8 doesn't appear to be compatible with SQLAlchemy 1.2.
+          CKAN 2.9 is, however.  To properly evaluate the risk,
+          we should examine all use of 'group_by' and 'order_by'
```
Comment (adborden) [inline .snyk:27]: @jbrown-xentity performed this analysis in CKAN 2.3 and no fixes were necessary. Maybe you can check with him on how to validate for 2.8.

File: .snyk [hunk 2]
```diff
@@ -1,10 +1,54 @@
...
+          to determine that all input to those operators are sanitized.
+        expires: 2020-09-10T06:00:00.000Z
+
+  SNYK-PYTHON-PASSLIB-569603:
+    - '*':
+        reason: >-
+          Fix is created but not yet published
```
Comment (adborden) [inline .snyk:34]: Have we done any analysis on the risk?

File: .snyk [hunk 3]
```diff
@@ -1,10 +1,54 @@
...
+        expires: 2020-11-19T06:00:00.000Z
+        
+  SNYK-PYTHON-WEBOB-40490:
+    - '*':
+        reason: >-
+          No remediation path available for CKAN2.8 (not compatible
+          with webob > 1.0.8).
```
Comment (adborden) [inline .snyk:41]: Have we done any analysis on the risk?

File: .snyk [hunk 4]
```diff
@@ -1,10 +1,54 @@
...
+        expires: 2020-11-19T06:00:00.000Z
+
+  SNYK-PYTHON-IPADDRESS-590065:
+    - '*':
+        reason: >-
+          No remediation available yet.  To properly evaluate the risk
+          we should examine the Python cryptography package and assess
+          whether and how it uses the hash() method of the classes
+          IPv4Interface and IPv6Interface.
```
Comment (adborden) [inline .snyk:50]: Can we perform the analysis described?

**Comment by woodt** [comment]
Have created risk assessment tickets per @adborden , so I think this can be merged.  So I shall!

**Pull request merged**